static/js/components/Navbar.js

```diff
@@ -13,6 +13,7 @@ import ProgramSelector from './ProgramSelector';
 import UserMenu from '../containers/UserMenu';
 
 const PROFILE_SETTINGS_REGEX = /^\/profile\/?|settings\/?|learner\/[a-z]?/;
+const PROFILE_PAGE_REGEX = /^\/profile\/[a-z]?/;
```

Comment:
what is `[a-z]` for in this regex? I think the only url we care about is `/profile`, right?